src/PurchaseHistory.js

```diff
@@ -0,0 +1,44 @@
+import React, { Component } from 'react';
+import Catelogue from './Catelogue.js';
+import backend from './backend/firebase-backend.js';
+
+class PurchaseHistory extends Component {
+
+    constructor() {
+        super();
+        this.state = { purchasedItems: [] }
+    }
+
+    componentWillMount() {
+        if (this.props.buyerID) this._getPurchaseHistory(this.props.buyerID);
+    }
+
+    _getPurchaseHistory = buyerID => {
+        backend.allItemsBought(buyerID)
+            .then(listingIDs => {
+                let arrItemObjs = []
+                listingIDs.forEach(ID => {
+                    backend.getItemDescription(ID)
+                        .then(item => {
+                            const itemObj = { productID: ID, blurb: item.blurb, image: item.image, price: item.price }
+                            arrItemObjs.push(itemObj);
+                            this.setState({ purchasedItems: arrItemObjs });
+                        })
+                })
+            })
+    }
+    render() {
+
+        return (
+            <div className="purchased-items-container">
```

Comment:
I'd add a title on this page so the shopper knows they're on their purchase history page. Maybe add a `<h2>Here's your purchase history</h2>` or something like that.